dashboard/app/models/regional_partner.rb

```diff
@@ -194,7 +194,7 @@ def self.find_by_zip(zip_code_raw)
             # Geocoder can raise a number of errors including SocketError, with a common base of StandardError
             # See https://github.com/alexreisner/geocoder#error-handling
             Retryable.retryable(on: StandardError) do
-              state = Geocoder.search(zip_code)&.first&.state_code
+              state = Geocoder.search(zip_code).select {|result| result.country_code.downcase == "us"}&.first&.state_code
```

Comment:
Is country code always non-nil (and does a result always have a country code)?

Reply:
ooh good point, I will add checks!